### PR TITLE
KOGITO-3009 Consolidate Docker image tests to use kogito-test-utils

### PR DESCRIPTION
**Pull request by cristianonicolai**
Depends on:
- https://github.com/kiegroup/kogito-runtimes/pull/692
- https://github.com/kiegroup/kogito-apps/pull/385

### CODE REVIEW

File: trusty/trusty-service/src/test/resources/application.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 trusty.explainability.enabled=false
 
 # Infinispan
-quarkus.infinispan-client.server-list=localhost:11232
+#quarkus.infinispan-client.server-list=localhost:11232
```
Comment (tiagodolphine) [inline trusty/trusty-service/src/test/resources/application.properties:5]: can't it be removed?

**Review by tiagodolphine — APPROVED** [review]
Great, LGTM

**Review by jstastny-cz — APPROVED** [review]

File: trusty/trusty-service/src/test/resources/application.properties (hunk 1 above)
Comment (cristianonicolai) [reply trusty/trusty-service/src/test/resources/application.properties:5]: I left in there just to make easier to config locally... but I dont mind removing if we think is better

File: pom.xml [hunk 2]
```diff
@@ -30,7 +30,6 @@
 
     <!-- containers used for testing -->
     <container.image.infinispan>quay.io/infinispan/server:${version.org.infinispan.image}</container.image.infinispan>
-    <container.image.keycloak>quay.io/keycloak/keycloak:${version.org.keycloak.image}</container.image.keycloak>
```
Comment (Sgitario) [inline pom.xml:33]: Why did you delete this property? As far I see, this property is expected in some modules :/

**Review by Sgitario — APPROVED** [review]
lgtm!

File: pom.xml (hunk 2 above)
Comment (cristianonicolai) [reply pom.xml:33]: its already defined in the parent pom, kogito-runtimes.
